build.js

```diff
@@ -55,7 +55,9 @@ rimraf("./builds", function() {
             fs: "empty"
         },
         resolve: {
-            root: ["./source"]
+            root: [
+                path.resolve("./source")
+            ]
```

Comment:
This is what allows us to use absolute paths like "index.js" instead of only relative paths like "./index.js". An absolute path is much easier to refactor.